spec/models/article_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe Article, type: :model do
+  before(:all) do
```

Comment:
With a larger test file, it's unlikely you're going to want to create the same `article` record for each of your model tests.

A better approach would be to use `let` to create this `article` variable: https://www.betterspecs.org/#let